microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -63,17 +63,17 @@ You use the Unity game engine and Visual Studio to create your HoloLens 2 projec
 **Unity**, a versatile game engine, is a popular choice for creating 3D applications, including mixed reality experiences for HoloLens 2 devices, due to its extensive features and robust capabilities for augmented reality (AR), virtual reality (VR), and mixed reality (MR).
 
 
+<!-- ------------------------------ -->
+###### Mixed Reality Feature Tool (MRFT)
+
+The **Mixed Reality Feature Tool (MRTK)** is a new way for developers to discover, update, and add Mixed Reality feature packages into Unity projects. You can search packages by name or category, see their dependencies, and even view proposed changes to your projects manifest file before importing. A manifest file is a JSON file defining all of your project's packages. Once you've validated the packages to be included, the Mixed Reality Feature tool will automatically download them into your selected Unity project.
```

Comment:
```suggestion
The **Mixed Reality Feature Tool (MRFT)** is a new way for developers to discover, update, and add Mixed Reality feature packages into Unity projects. You can search packages by name or category, see their dependencies, and even view proposed changes to your projects manifest file before importing. A manifest file is a JSON file defining all of your project's packages. Once you've validated the packages to be included, the Mixed Reality Feature tool will automatically download them into your selected Unity project.
```
maybe add this point that got dropped:
Installs the **Mixed Reality Toolkit (MRTK)** and the **Microsoft Mixed Reality WebView plugin for Unity**.